src/cloud_what/providers/azure.py

```diff
@@ -146,6 +150,16 @@ def get_api_versions(self) -> Union[List[str], None]:
                 api_versions = api_versions_dict['apiVersions']
         return api_versions
 
+    def _has_azure_linux_agent(self):
```

Comment:
I would like to see here some doc string.

Reply:
okay.